internal/releases/team.go

```diff
@@ -169,9 +169,9 @@ func (t *Team) populateGithubRepos(org config.GithubOrg) error {
 		}
 		wg.Wait()
 
-		// Iterate over repos, add only those that have releases to the Team's list of repos
+		// Iterate over repos and add the unarchived ones that have at least one commit
 		for _, r := range repos {
-			if len(r.Info().Releases) > 0 {
+			if !r.Info().IsArchived && (len(r.Info().Releases) > 0 || len(r.Info().Commits) > 0) {
 				t.info.Repos = append(t.info.Repos, r.Info())
```

Comment:
Huh - reading this my comment about not processing archived repos is probably void, given that archived repos should never get added for processing? That means you can probably just remove the IsArchived field because it's implied true surely?